styles/scss/components/progress.scss

```diff
@@ -53,8 +53,7 @@
 .video-react .video-react-mouse-display {
   @extend .video-react-icon;
 
-  &:before {
-    @extend .video-react-icon-circle:before;
+  .video-react-icon-circle:before, &:before {
```

Comment:
I'm confused.

Why did it need to extend this style when it's hidden? @JimLiu 

Why did you make this change? @xiaoyuhen

Reply:
should not change the css file

Reply:
Looks like this whole `.video-react .video-react-mouse-display` block is not really used. I tried to remove it and `MouseTimeDisplay` component still looks good. Let's just get rid of it.

Reply:
already remove this style